src/rubicon/objc/types.py

```diff
@@ -889,6 +915,12 @@ class CFRange(Structure):
         ("length", CFIndex),
     ]
 
+    def __repr__(self):
+        return f"<CFRange({self.location}, {self.length})>"
```

Comment:
As with insets, there's no "natural" order or label here, so labelling them as "location=123, length=456" would be helpful